windows.md

```diff
@@ -54,22 +54,36 @@ With those compatibility things out of the way, you're ready to start the system
    ```bash
    choco install slack -y
    ```
-7. Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
+7. Close PowerShell and open it again as administrator (like in step 2). Copy the following text and right-click in the blue middle part of the PowerShell window to paste the text. Hit enter.<br><br>
```

Comment:
We need to reopen the terminal because, after installing with choco, the environment variables changed, as explained in this [comment](https://github.com/upleveled/system-setup/pull/91#issuecomment-2270597504).

Reply:
[the linked comment](https://github.com/upleveled/system-setup/pull/91#issuecomment-2270597504) doesn't explain a reason why we should move the "close and reopen"  from step 8 to step 7 in the Windows guide

the comment explains that `node -v` didn't work, but that's not a command that we run in step 7

check into this again - if we actually have a problem with `corepack` being unavailable, then of course we can make this or other changes. but I'm guessing that we don't have an issue here.

Reply:
I checked it on Windows again, removed `nodejs-lts` from my system with all folders, also `pnpm`, and started new with the installation of `node`. For the installation, I used the same commands as the System Setup Guide as seen in the first screenshot. We can see the last command where I used `corepack enable`, and we get an error message that `corepack` is not recognized. 
![corepack-missing](https://github.com/user-attachments/assets/d026b7d8-209e-4aec-b29c-e92aea2a685e)

After restarting the terminal, `corepack` is recognized and I can use the latest `pnpm` version to install Preflight
![corapack-restart](https://github.com/user-attachments/assets/c039d221-9462-4ebc-8650-684b6cd87c01)

Reply:
removing the "Close PowerShell and open it again ..." from the following step may introduce a new error:

- https://github.com/upleveled/system-setup/pull/91#discussion_r1726895994